projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -109,7 +109,7 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
             {{ 'claim.subclaim.statusDescription' | cxTranslate }}
           </div>
           <div class="col-6 col-sm-5 value notice">
-            {{ subclaim.status }}
+            <!-- {{ display statusDescription }} -->
```

Comment:
I left this as a reminder. We aren't sure what is statusDescription since there isn't such attribute in the model.